src/Mutation/PostObjectCreate.php

```diff
@@ -5,345 +5,358 @@
 use GraphQL\Deferred;
 use GraphQL\Error\UserError;
 use GraphQL\Type\Definition\ResolveInfo;
+use GraphQLRelay\Relay;
 use WPGraphQL\AppContext;
 use WPGraphQL\Data\DataSource;
 use WPGraphQL\Data\PostObjectMutation;
 
 class PostObjectCreate {
-    /**
```

Comment:
This is hard to see what changed as it's showing the entire file was changed instead of just a few lines. My guess is because of your IDE settings formatting it differently?